front_end/src/components/onboarding/OnboardingSettings.ts

```diff
@@ -0,0 +1,42 @@
+export const onboardingTopics = [
+  {
+    name: "US Politics",
```

Comment:
What's the longer term plan for these? For now they are hard-coded in the code, but in a few weeks/months will become less relevant and will need to be updated. Do we want to move them to the backend to be changed in the admin, or?

Reply:
Yes, the plan is to make these editable via admin. It shouldn't block launch, but the earlier we implement that would be better.